app/components/autocomplete/at_mention/at_mention.js

```diff
@@ -64,6 +65,10 @@ export default class AtMention extends PureComponent {
     }
 
     componentDidUpdate(prevProps, prevState) {
+        if (this.props.appsTakeOver) {
```

Comment:
if this is true, do you want to execute the action on line 60?

Reply:
I don't think so. The apps have taken over, and the user searches are done by the apps.